components/antlib/scripts/parse_version

```diff
@@ -82,7 +82,7 @@ def snapshot_logic(omero_version):
     if m:
         # Non-tag version
         base = incr_version(m.group("BASE"))
-        omero_version = "%s%s-SNAPSHOT" % (
```

Comment:
Should this change be reverted if the `-ice` separator is maintained?

Reply:
well spotted